django_restql/fields.py

```diff
@@ -60,14 +60,12 @@ def validate_pk_list(self, pks):
             return validator.run_validation(pks)
     
         def validate_data_list(self, data, partial=False):
-            request = self.context.get('request')
-            context = {"request": request}
             model = self.parent.Meta.model
-            rel = getattr(model, self.source).rel
+            rel = getattr(model, self.field_name).rel
 
             if isinstance(rel, ManyToOneRel):
                 # ManyToOne Relation
-                field_name = getattr(model, self.source).field.name
+                field_name = getattr(model, self.field_name).field.name
```

Comment:
Not sure if this is needed since we have `self.field_name`. Can `field.name` be different?

Reply:
You mean if `getattr(model, self.source).field.name` will be different from `getattr(model, self.source)`?

> @yezyilomo That makes a lot of sense. If this gets in I'll refactor how we check for the query param and get parsed values on that view mixin. Do you prefer I rebase or merge?

Just merge.

Reply:
Gotcha. And I agree that source is different, but I mean is `getattr(model, self.field_name).field.name` different from `self.field_name`?

Reply:
Yeah it's different.